preprocessing/nextclade/src/loculus_preprocessing/processing_functions.py

```diff
@@ -68,6 +69,44 @@ def warn_potentially_invalid_authors(authors: str) -> bool:
     return bool(";" not in authors and len(authors_split) > 3)  # noqa: PLR2004
 
 
+def reformat_authors_from_latin_to_ascii(authors: str) -> str:
+    ascii_authors = unicodedata.normalize("NFKD", authors).encode("ascii", "ignore").decode("ascii")
+    return ascii_authors
+
+
+def check_latin_characters(
+    authors: str, input_fields: list[str], output_field: str
+) -> tuple[list[ProcessingAnnotation], list[ProcessingAnnotation]]:
+    warnings: list[ProcessingAnnotation] = []
+    errors: list[ProcessingAnnotation] = []
+    counter = 0
+    # Check if all characters in the authors string are Latin letters or spaces (transformable to ASCII)
+    for char in authors:
+        # If character is already ASCII, skip
+        if ord(char) < 128:
+            continue
+        # Normalize the character (NFKD) and attempt to encode to ASCII, ignoring diacritics.
+        decomposed = unicodedata.normalize("NFKD", char)
+        ascii_equiv = decomposed.encode("ascii", "ignore").decode("ascii")
```

Comment:
Wouldn't more sense to run `decomposed.encode("ascii")`  (without  `ignore`) and catch the  error, rather than ignoring the error and then checking whether result the is empty (which would be better as explicit check, not relying on empty strong evaluating to `False` IMO)